docs/how-to/h-benchmark.md

```diff
@@ -0,0 +1,51 @@
+# How to benchmark Charmed Apache Kafka
+The Apache Kafka benchmark charm uses the OpenMessaging tool to test both producer and consumer performance in the cluster.
+
+The OpenMessaging allows for a distributed deployment, where the charm leader will run the main "manager" process and gather the metrics, whilst other units will act as followers and act as producer/consumers of the cluster.
+
+## Deploy
+Create a new Juju model:
+```
+juju add-model kafka-benchmark
+Deploy Apache Kafka with Juju:
```

Comment:
This doesn't look like a command.
**todo**: fix the code blocks to contain only commands.